Tests/Parser/Client/fixtures/mobile_app.yml

```diff
@@ -275,3 +275,45 @@
     type: mobile app
     name: Kik
     version: 15.15.0.21560
+-
+  user_agent: 1PasswordThumbs/1 CFNetwork/596.5 Darwin/12.5.0 (x86_64) (MacBookAir4%2C1)
+  client:
+    type: mobile app
+    name: 1Password
+    version: "1"
+-
+  user_agent: Word/15.22.506 (Mac OS/10.11.5; Desktop; en-US; NonAppStore; Apple/MacBookAir7,1)
+  client:
+    type: mobile app
+    name: Microsoft Office Word
+    version: 15.22.506
+-
+  user_agent: Microsoft Office PowerPoint/15.29.1215 (Mac OS/10.11.5; Desktop; en-US; NonAppStore; Apple/MacBookAir7,2)
+  client:
+    type: mobile app
+    name: Microsoft Office PowerPoint
+    version: 15.29.1215
+-
+  user_agent: Microsoft Office Word/16.11.319 (Mac OS/10.10.5; Desktop; en-US; NonAppStore; Apple/MacBookAir6,1)
+  client:
+    type: mobile app
+    name: Microsoft Office Word
+    version: 16.11.319
+-
+  user_agent: Microsoft Office Excel/16.44.1213 (Mac OS/11.1; Desktop; en-US; AppStore; Apple/MacBookAir10,1)
+  client:
+    type: mobile app
+    name: Microsoft Office Excel
+    version: 16.44.1213
+-
+  user_agent: Microsoft Office OneNote/16.43/201106 (iOS/14.0.1; Tablet; es-MX; AppStore; Apple/iPad11,6)
```

Comment:
here, too, you can define the device in the next PR

regex: `Apple/iPad11,6`
model: 'iPad 8 10.2" (WiFi)'

I think this list might help  https://everyi.com/by-identifier/ipod-iphone-ipad-specs-by-model-identifier.html